.travis.yml

```diff
@@ -94,6 +92,12 @@ before_install:
 
       # Qt 5
       PATH=/opt/qt55/bin:${PATH}
+
+      if [ "$build_system" = "cmake" ]; then
+          COMPILER_VERSION=6
+          sudo update-alternatives --install /usr/bin/${CC} ${CC} /usr/bin/${CC}-${COMPILER_VERSION} 60 --slave /usr/bin/${CXX} ${CXX} /usr/bin/${CXX}-${COMPILER_VERSION}
+          export CXX="${CXX}-${COMPILER_VERSION}" CC="${CC}-${COMPILER_VERSION}"
```

Comment:
Copied this from another project, and now I don't remember why did I use both switches (env var and update-alternatives). Here it seems to be working just fine with environment variables only.